docs/running-on-yarn.md

```diff
@@ -304,6 +304,16 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.am.attemptFailuresValidityInterval</code></td>
+  <td>(none)</td>
+  <td>
+  Defines the validity interval (in millisecond) for AM failure tracking.
```

Comment:
You shouldn't say "in milliseconds". The user can specify the unit in the value (e.g. "1d" or "4500ms").